src/hubbard/hubbard_utils.cpp

```diff
@@ -0,0 +1,215 @@
+#include <cstdio>
+#include <cstdlib>
+#include <string>
+#include <vector>
+#include <tuple>
+#include "context/simulation_context.hpp"
+
+namespace sirius {
+
+static std::vector<std::string>
+split(std::string str, std::string token)
```

Comment:
Check if this function can help
https://github.com/electronic-structure/SIRIUS/blob/develop/src/core/string_tools.hpp#L32

Reply:
yes it will. I will use it.